library/src/main/java/com/github/gfx/android/orma/SingleAssociation.java

```diff
@@ -31,7 +35,7 @@
  * @param <Model> The type of a model to relate.
  */
 @JsonAdapter(SingleAssociationTypeAdapterFactory.class)
-public class SingleAssociation<Model> {
+public class SingleAssociation<Model> implements Parcelable {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Generics names should be a one letter long and upper case.](https://www.codacy.com/app/gfuji/Android-Orma/file/2039516422/issues/source?bid=3128773&fileBranchId=3130085#l38)